Makefile

```diff
@@ -42,13 +46,14 @@ commit-hook:
 full-clean: check-docker
 	docker images -q $(IMAGE_PREFIX)$(COMPONENT) | xargs docker rmi -f
 
-postgres:
-	docker start postgres || docker run --restart="always" -d -p 5432:5432 --name postgres postgres:9.3
-	docker exec postgres createdb -U postgres deis 2>/dev/null || true
-	@echo "To use postgres for local development:"
-	@echo "    export PGHOST=`docker-machine ip $$(docker-machine active) 2>/dev/null || echo 127.0.0.1`"
-	@echo "    export PGPORT=5432"
-	@echo "    export PGUSER=postgres"
+start-postgres:
+	docker run -d --name deispg postgres:9.4
+	# wait for the database to come up
+	sleep 3
+
+stop-postgres:
+	docker stop deispg
```

Comment:
a minor DRY improvement: might consider defining a function in this Makefile executing these commands:
```
define stop-pg
  	docker stop deispg && docker rm deispg
endef
```
and then every target that needs to run this logic can then do so:
```
stop-postgres:
  $(call stop-pg)
...
do-unit-test:
  docker  run ... || $(call stop-pg)
```